### ISSUE TITLE
Remove "View Classic Desktop Site" from footer

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#3352]
If we don't provide an escape hatch to the old Django site we should remove the "View Classic Desktop Site" link/action from the footer.

To be clear I think this is a good idea 👍

**Comment by muffinresearch** [comment]
Whilst we will need this - it's too early to do this now.

**Comment by tsl143** [comment]
Ohkay, so shall I close this request or just leave as it is.
Though I want to push TestFooter.js as I rewrote it using enzyme

**Comment by muffinresearch** [comment]
@tsl143 if you want to split that out into a separate patch we can merge handle it independently.

**Comment by tsl143** [comment]
@muffinresearch sure thing :)

**Comment by tsl143** [comment]
> if you want to split that out into a separate patch we can merge handle it independently.

Created a PR for this https://github.com/mozilla/addons-frontend/pull/3361

**Comment by tofumatt** [comment]
I don't like this issue hanging around while it's blocked on an unclear list of things. The change itself to remove the link is pretty simple; I think we should either re-open this when ready, file new issue when ready, or have a checklist of issues that need to be closed before this one can be merged.

For now, I'ma close this one 😅. Please re-open or re-file when ready 😄
